python/ray/dag/tests/experimental/test_multi_node_dag.py

```diff
@@ -97,10 +97,10 @@ def _get_node_id(self) -> "ray.NodeID":
         z = c.inc.bind(inp)
         dag = MultiOutputNode([x, y, z])
 
-    adag = dag.experimental_compile()
+    cdag = dag.experimental_compile()
```

Comment:
do we want `cdag` or `cgraph` ?

Reply:
external API: `cgraph`
internally, `cgraph` and `cdag` can be used interchangeably
here `cdag` is easier to read since we call `experimental_compile()` on `dag`